packages/ui-components/src/components/action-button/usage/angular.md

```diff
@@ -0,0 +1,54 @@
+```html
+<!-- Primary -->
+<kv-action-button
+	text="Primary Button"
+	type="primary"
+></kv-action-button>
+
+<!-- Secondary -->
+<kv-action-button
+	text="Secondary Button"
+	type="secondary"
+></kv-action-button>
+
+<!-- Tertiary -->
+<kv-action-button
+	text="Tertiary Button"
+	type="tertiary"
+></kv-action-button>
+
+<!-- Disabled -->
+<kv-action-button
+	text="Disabled Button"
+	type="primary"
+	disabled="true"
+></kv-action-button>
+
+<!-- Small -->
+<kv-action-button
+	text="Small Button"
+	type="primary"
+	size="small"
+></kv-action-button>
+
+<!-- With icon -->
+<kv-action-button
+	text="Small Button"
+	type="primary"
+	icon="kv-add"
+></kv-action-button>
+
+<!-- Icon only -->
+<kv-action-button
+	type="primary"
+	icon="kv-add"
+></kv-action-button>
+
+<!-- Split icon -->
+<kv-action-button
+	text="Split icon Button"
+	type="primary"
+	icon="kv-add"
+	split-icon="kv-arrow-drop-down"
```

Comment:
Just a small note: in angular and react we can use the actual prop name instead of the html attribute, so we could reflect this in our usage examples.